### DISCUSSION TITLE
Artisan not starting on a raspberryPI

### DISCUSSION THREAD

**Discussion by KrazeyNKrusty** [artisan-roaster-scope/artisan#1328] [General]
Just completed a fresh install of raspberry Pi OS - Bookworm.  I believe this is the last version that will run on the Raspberry PI 2.

I have been able to connect to the PI via VNC over WIFI.

I was able to download and install artisan-linux-2.10.0_raspbian-bookworm.deb

The artisan software is listed under Accessories.  When I attempt execute the software, I get an hourglass mouse pointer for about 20 seconds before it returns to normal cursor.  The application does not start.

Is there a log file or similar that I can examine for error messages?

Regards
Paul

**Comment by MAKOMO** [comment]
The Artisan build is for 64bit installations only and most likely does not support the RPi2. Sorry.

**Reply by KrazeyNKrusty** [reply]
Thank you.
